controllers/buildpipeline/buildpipeline_controller.go

```diff
@@ -92,17 +92,20 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 				return ctrl.Result{}, err
 			}
 		}
-		return helpers.HandleLoaderError(logger, err, "Component", "PipelineRun")
+		return helpers.HandleLoaderError(logger, err, "component", "pipelineRun")
+	} else if component == nil {
```

Comment:
Looks like we're missing coverage for this condition, it would probably be good to add an unit test that checks that we don't error out for a pipelineRun without the component label.

Reply:
Nevermind, we already have a test for that, I misread the codecov report.